### PR TITLE
Add docs for the -OutFile switch on New-TestResources.ps1

### PR DESCRIPTION
**Pull request by christothes**

### CODE REVIEW

File: eng/common/TestResources/New-TestResources.ps1.md [hunk 1]
```diff
@@ -412,6 +412,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OutFile
```
Comment (heaths) [inline eng/common/TestResources/New-TestResources.ps1.md:415]: This needs to be added to the bottom of the .ps1 file, if not already. This shouldn't be hand-authored. Let me know if you need help generating the doc file (there's a module you need, and need to hack the help a bit afterward).

Comment (heaths) [reply eng/common/TestResources/New-TestResources.ps1.md:415]: Or, if you'd rather, just wait till I make some updates anyway to defaulting this switch for our repo, in which case I'll be regenerating docs anyway.

Comment (christothes) [reply eng/common/TestResources/New-TestResources.ps1.md:415]: It is in the PS1 already - any pointers to generating the doc would be helpful, thanks!

Comment (heaths) [reply eng/common/TestResources/New-TestResources.ps1.md:415]: Then just leave this for now. I'll rerun it again later.

**Pull request merged**